app/assets/stylesheets/lux/_show_button.scss

```diff
@@ -0,0 +1,46 @@
+@import "lux/variables";
```

Comment:
Could you please convert the larger pixel values in this file to use rems?

Reply:
This has been addressed and in the next commit.